src/Rings/mpoly-ideals.jl

```diff
@@ -11,7 +11,7 @@ Given a vector `V` of polynomials in `R`, return the ideal of `R` generated by t
 
 # Examples
 ```jldoctest
-julia> R, (x, y) = polynomial_ring(QQ, [:x, :y])
+julia> R, (x, y) = polynomial_ring(QQ, ["x", "y"])
```

Comment:
this one and the similar changes should not get changed back (compared to https://github.com/oscar-system/Oscar.jl/pull/4133).